### ISSUE TITLE
[Bug]: Doubled menu bar in 2023.3

### ISSUE THREAD

**Issue by maicol07** [mallowigi/material-theme-issues#428]
### Describe the bug

The menu bar is doubled (see screenshot) when reopening the IDE with the last project and "Reopen last opened project" setting is checked in IDE settings.

### To Reproduce

1. Open a project
2. Close the IDE (be sure that "Reopen last project when the IDE is opened" is checked in the settings)
3. Reopen the IDE

### Expected behavior

Menu bar isn't doubled

### Material Theme Version

8.13.2

### IDE

PHPStorm

### System Info

```shell
PhpStorm 2023.3 Beta
Build #PS-233.11799.21, built on November 13, 2023
Licensed to PhpStorm EAP user: Maicol Battistini
Expiration date: December 13, 2023
Runtime version: 17.0.9+7-b1087.3 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Linux 5.15.133.1-microsoft-standard-wsl2
GC: G1 Young Generation, G1 Old Generation
Memory: 6452M
Cores: 24
Registry:
  ide.balloon.shadow.size=0
  editor.focus.mode.color.light=d2d4d5
  editor.focus.mode.color.dark=d2d4d5
  debugger.new.tool.window.layout=true
  run.processes.with.pty=TRUE
  ide.experimental.ui=true
  ide.editor.tab.selection.animation=true
Non-Bundled Plugins:
  com.chrisrm.idea.MaterialThemeUI (8.13.2)
Current Desktop: Undefined
```

### Logs/Screenshots
![image](https://github.com/mallowigi/material-theme-issues/assets/9463142/7823e04a-139b-4566-aed0-7b6f7d31e502)

**Comment by mallowigi** [comment]
Doesnt' happen on Mac. I'll try to find a linux machine asap, although I'm not sure what can be the cause.

**Comment by nadia-tarashkevich** [comment]
FYI: it also happens on Windows
https://youtrack.jetbrains.com/issue/IDEA-340566/Sometimes-duplicate-toolbar-appear-after-updated-to-2023.3
Please let us know, if you need a hand in investigation
Nadezhda Tarashkevich
JetBrains Support 

<blockquote><img src="static/favicon.ico?3c05e1a5b709165a5c4e825adf27273a?3c05e1a5b709165a5c4e825adf27273a" width="48" align="right"></blockquote>

**Comment by alvaro-canepa** [comment]
On linux, it's happens opening a project from toolbox window. Doesn't happens from phpStorm window projects.

https://youtrack.jetbrains.com/issue/IDEA-341774/Double-toolbar-on-open-project

**Comment by kurtmisc** [comment]
Happens in pycharm, but only for one of my projects. The other projects seem to be fine and to resolve I close the project with the problem and reopen from a project that is displaying correctly.

**Comment by Juje** [comment]
Same here on Ubuntu 23.10 with IntelliJ IDEA Ultimate.

**Comment by mallowigi** [comment]
Is it still happening ? I tried on Windows and Linux and I can't reproduce either.

**Comment by alvaro-canepa** [comment]
@mallowigi Is fixed in the latest PhpStorm 2024.1 ( Build #PS-241.14494.237, built on March 27, 2024 )

**Comment by mallowigi** [comment]
Cool
